health/health_config.c

```diff
@@ -45,6 +47,11 @@ static inline int rrdcalc_add_alarm_from_config(RRDHOST *host, RRDCALC *rc) {
     if (rrdcalc_exists(host, rc->chart, rc->name, rc->hash_chart, rc->hash))
         return 0;
 
+    if ( (!rc->module_match && rc->plugin_match) || (rc->module_match && !rc->plugin_match)) {
+        error("Health configuration for alarm '%s.%s' has neither \"module\" nor \"plugin\", but it is necessary to define both. Ignoring it.", rc->chart, rc->name);
+        return 0;
+    }
```

Comment:
i suggest to remove the check, it could make no sense but it is not prohibited, it is valid config

Reply:
This sentence looks like contradictory with your comment [here](https://github.com/netdata/netdata/pull/10041#discussion_r501217373). :smile:

Reply:
It is not:
- could make no sense
- not prohibited


=> you do not prohibit it but make the example meaningful

It is not mandatory to have both `plugin` and `module` set. At least i see no reason in doing so.